wordle-web/app.vue

```diff
@@ -1,17 +1,29 @@
 <template>
   <NuxtLayout>
     <v-app>
-      <v-app-bar color="primary" :elevation="2">
+      <v-app-bar color="primary">
+        
+        <v-app-bar-title>Pentagram</v-app-bar-title>
         <template v-slot:prepend>
-          <v-icon color="secondary"> mdi-book </v-icon>
+          <v-icon size="x-large">mdi-pentagram</v-icon>
```

Comment:
I like that ya'll kept the pentagram theme lol

Reply:
absolutely, I tried messing with the font too